src/app/static/src/app/store/dataExploration/dataExploration.ts

```diff
@@ -0,0 +1,85 @@
+import {Language} from "../translations/locales";
+import {currentHintVersion} from "../../hintVersion";
+import {hintrVersion, HintrVersionState, initialHintrVersionState} from "../hintrVersion/hintrVersion";
+import {adr, ADRState, initialADRState} from "../adr/adr";
+import {genericChart, GenericChartState, initialGenericChartState} from "../genericChart/genericChart";
+import {ADRUploadState, initialADRUploadState} from "../adrUpload/adrUpload";
+import {baseline, BaselineState, initialBaselineState} from "../baseline/baseline";
+import {initialMetadataState, metadata, MetadataState} from "../metadata/metadata";
+import {
+    initialSurveyAndProgramState,
+    surveyAndProgram,
+    SurveyAndProgramState
+} from "../surveyAndProgram/surveyAndProgram";
+import {
+    initialPlottingSelectionsState,
+    plottingSelections,
+    PlottingSelectionsState
+} from "../plottingSelections/plottingSelections";
+import {errors, ErrorsState, initialErrorsState} from "../errors/errors";
+import {localStorageManager} from "../../localStorageManager";
+import {Module, StoreOptions} from "vuex";
+import {TranslatableState} from "../../root";
+import {Error} from "../../generated";
+import {actions} from "./actions";
+import {mutations} from "./mutations";
+import {getters} from "./getters";
+
+declare const currentUser: string;
+
+export interface DataExplorationState extends TranslatableState {
```

Comment:
Might want to call this something else, `SharedState` or `BaseRootState` or something to reflect the fact that the main app `RootState` inherits from this.